packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -74,6 +107,16 @@ export default function Sidebar({
     [sidebarExpanded, sidebarRendered, activePage],
   );
 
+  useEffect(() => {
+    if (squadVersion !== SquadVersion.Off) {
```

Comment:
In case the user is not logged in, but squad is on it will get tracked.
My suggestion is either create a component specifically for this button with the proper tracking.
Or have a boolean here in the sidebar that represents whether the squad button should be visible or not

Reply:
I will go with the boolean value, as because of the placement we can't use one generic component.